docs/handbook/image-file-formats.rst

```diff
@@ -552,12 +552,13 @@ JPEG 2000
 
 .. versionadded:: 2.4.0
 
-Pillow reads and writes JPEG 2000 files containing ``L``, ``LA``, ``RGB`` or
-``RGBA`` data.  It can also read files containing ``YCbCr`` data, which it
-converts on read into ``RGB`` or ``RGBA`` depending on whether or not there is
-an alpha channel.  Pillow supports JPEG 2000 raw codestreams (``.j2k`` files),
-as well as boxed JPEG 2000 files (``.j2p`` or ``.jpx`` files).  Pillow does
-*not* support files whose components have different sampling frequencies.
+Pillow reads and writes JPEG 2000 files containing ``L``, ``LA``, ``RGB``,
+``RGBA``, or ``YCbCr`` data.  When reading, ``YCbCr`` data is converted to
+``RGB`` or ``RGBA`` depending on whether or not there is an alpha channel.
+Beginning with version 8.3.0, Pillow can read ``RGB``, ``RGBA``, and
+``YCbCr`` images whose components have different sampling frequencies.
```

Comment:
Two notes:

* I'm not sure this needs to be stated in the documentation at all, since it is what I would expect by default (I didn't see this text when writing #4996 and thought of it as a bugfix, not a new feature).
* AFAIK, Pillow<8.3.0 cannot read images with any subsampling, not just if it is different for some components (although I couldn't find any images where all components are subsampled to test this).

Reply:
Well, it's a polite way to say "before 8.3.0, Pillow didn't support subsampling".  Also, there are still limitations that should be documented: subsampled grayscale images aren't supported (I guess those are unlikely in practice) and Pillow can't subsample on write.

Updated to fix the implied historical behavior and to make the write limitation explicit.